install/gcp/gcp_installer.py

```diff
@@ -62,6 +62,9 @@ def run():
     parser.add_argument('--no-cloudshell',
                         action='store_true',
                         help='Bypass Cloud Shell requirement')
+    parser.add_argument('--service-account-key-path',
```

Comment:
Please see my comment below about changing this to `--service-account-key-file`.

Reply:
Done